recipes-extended/ostree/ostree-2024.4/0001-deploy-Ensure-boot-directory-is-open-before-accessin.patch

```diff
@@ -0,0 +1,32 @@
+From 3f8dba21c5913b09e36f76215e77b7d6e7816453 Mon Sep 17 00:00:00 2001
+From: Rogerio Guerra Borin <rogerio.borin@toradex.com>
+Date: Tue, 12 Mar 2024 17:02:58 -0300
+Subject: [PATCH] deploy: Ensure boot directory is open before accessing it
+
+This fixes a bug in the (early) deployment pruning function which before
+tried to access the boot directory without opening it first.
+
+Upstream-Status: Accepted
```

Comment:
I see this was accepted upstream as you stated here: https://github.com/ostreedev/ostree/commit/4efb44f35a099c54b9e2895cea9622a97ab6d638

I also see that OSTree just released version 2024.5 that appears to contain this commit: https://github.com/ostreedev/ostree/commits/v2024.5/

Just released ~7hrs ago at the time of me writing this. Would it be better to bump the OSTree version so we get this from upstream rather than having this patch here?

Reply:
I'd prefer to do the bump and remove the patch later, otherwise I'd need to retest everything with the new version.

Reply:
Alright fair enough then. I'll proceed with the merge then.